### ISSUE TITLE
Chinese characters not displaying properly in the plugin

### ISSUE THREAD

**Issue by kalulas** [groboclown/p4ic4idea#234]
**Describe the bug**
Encodings of both my device / P4Charset / IDE(I am using GoLand 2021.3.1) are set to UTF-8(no BOM), and all the Chinese characters are displaying correctly except in the plugin window of GoLand. It seems that all these characters are GBK-encoded.

**To Reproduce**
1. Create a changelist in P4
2. Write a description that contains Chinese characters, for example, `这是一段中文文本` (it means 'This is a text written in Chinese')
3. Launch or go back to Intellij IDEA and click the 'Refresh' button in the plugin view
4. The description is not displayed correctly and is not readable
5. FYI, this description is properly displayed in JetBrains Rider 2022.2.4, which uses its own built-in Perforce plugin I believe

**Expected behavior**
Descriptions containing Chinese characters are all readable in the plugin view.

**Attachments**
step2: ![issue-step2](https://user-images.githubusercontent.com/27734649/203483429-6918ded9-dbe2-4860-8a65-9af441517736.png)
step4: ![issue-step4](https://user-images.githubusercontent.com/27734649/203483381-49905a40-ff38-4b9e-ba28-0fb4c7a370df.png)
the same changelist in Rider: ![image](https://user-images.githubusercontent.com/27734649/203484313-76c73f68-1993-4d3b-93dc-1d3ef75759e4.png)
log(GoLand 2021.3.1): [idea.log](https://github.com/groboclown/p4ic4idea/files/10073148/idea.log)

**Desktop (please complete the following information):**
 - OS: Windows10 22H2
 - IDE and version: GoLand 2021.3.1
Build #GO-213.6461.23, built on December 16, 2021
 - Version [0.12.0]

**Comment by shiyunjin** [comment]
I fixed
At the bottom of the `goland.vmoptions` file add
```
-Dfile.encoding=utf-8
-Dsun.jnu.encoding=UTF-8
```
then restart goland

**Comment by kalulas** [comment]
> I fixed At the bottom of the `goland.vmoptions` file add
> 
> ```
> -Dfile.encoding=utf-8
> -Dsun.jnu.encoding=UTF-8
> ```
> 
> then restart goland

![image](https://user-images.githubusercontent.com/27734649/211288288-782dd3fe-925d-4ffb-aabe-f7e2ff73f34e.png)
It works for me! Thank you for your help 😄

**Comment by shiyunjin** [comment]
I don't think this issue should have been closed, it's a problem that needs to be fixed. Not everyone should have to do this manually.

**Comment by kalulas** [comment]
> I don't think this issue should have been closed, it's a problem that needs to be fixed. Not everyone should have to do this manually.

Agree, I have reopened this issue
